dts/common/nordic/nrf_common.dtsi

```diff
@@ -8,6 +8,7 @@
 #include <zephyr/dt-bindings/adc/nrf-adc.h>
 #include <zephyr/dt-bindings/gpio/gpio.h>
 #include <zephyr/dt-bindings/i2c/i2c.h>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
```

Comment:
nit: maybe split, it feels weird to see this in board commit (even though it's required)

Reply:
Done.